.gitignore

```diff
@@ -0,0 +1,2 @@
+src/StackTask/bin
```

Comment:
We have ready to use `.gitignore` file that you can use:
https://github.com/github/gitignore/blob/main/VisualStudio.gitignore